packages/cursor/init-gpg.sh

```
@@ -0,0 +1,105 @@
+chmod 700 "$GNUPGHOME"
+
+cat > "$GNUPGHOME/gpg-agent.conf" <<EOF
+allow-preset-passphrase
+default-cache-ttl 28800
+max-cache-ttl 86400
+EOF
+
+gpgconf --kill gpg-agent 2>/dev/null || true
+gpgconf --launch gpg-agent
+
+echo "$GPG_PRIVATE_KEY_BASE64" | base64 -d | gpg --batch --import 2>/dev/null
```

Shellcheck (0.11.0)

[style] Prefer putting braces around variable references even when not strictly required.

(SC2250)